src/main/java/br/com/invillia/lyon/userevents/service/UserService.java

```diff
@@ -0,0 +1,60 @@
+package br.com.invillia.lyon.userevents.service;
+
+import br.com.invillia.lyon.userapi.events.UserEvent;
+import br.com.invillia.lyon.userevents.Repository.PersonRepository;
+import br.com.invillia.lyon.userevents.domain.User;
+import br.com.invillia.lyon.userevents.mapper.UserMapper;
+import br.com.invillia.lyon.userevents.response.UserResponse;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+@Slf4j
+public class UserService {
+
+    @Autowired
+    private PersonRepository personRepository;
+
+    @Autowired
+    private UserMapper userMapper;
+
+    public void sendUpdate(UserResponse userResponse, String id) {
+        User user = userMapper.updateUser(userResponse, id);
+        saveUser(user);
+    }
+
+    public void saveUser(User user) {
+       Boolean check = searchAllUsersInTheBank(user.getId_user());
+
+       if(check != true){
```

Comment:
`check != true`  não é a mesma coisa de `!check`?

Reply:
:cry:

Reply:
A variavel Boolean já tem como default o false né

Reply:
Boolean por default é null, mas o que esse código vai printar?

```
Boolean teste = true;
System.out.println(!teste);
```

Reply:
Lodi esses println eu coloquei só pra ver se estava entrando no método pq tava dando um erro quando corrigi os primeiros PR mas eu fiz outro PR removendo eles se for isso q vc ta se referindo